railties/lib/rails/api/task.rb

```diff
@@ -92,19 +96,14 @@ def desc(description)
       end
 
       def load_and_configure_sdoc
-        require "sdoc"
```

Comment:
This `require` was down here to avoid bothering to load sdoc unless we're actually using it. Seems like we should keep the location, even if we're now loading our own file [which then loads sdoc in turn].

(Especially as the method's called "load_and_configure_sdoc" 😉)